### PR TITLE
feat(brand logo): add support for logo on title

### PR DESCRIPTION
**Pull request by f-fouad**
## Prerequisites

- [x] My Pull Request is filled against the `next` branch
- [X] All commits in my Pull Request conform to [Elegant Git commit Guidelines](https://elegant.oncrashreboot.com/git-commit-guidelines)

## Recommended Steps

- [X] My patch adds a new feature, therefore I have also added a help article about it
- [X] My patch changes Elegant behavior, therefore I have updated the help article to reflect this change
- [ ] My commits are [signed](https://help.github.com/en/articles/signing-commits)

## Description

Added an optional variable `DISPLAY_BRAND_LOGO`, if defined, a logo image will be displayed Left of the site title in the navigation bar. The variable's value should be the url to the brand logo file.

An example in pelicanconf.py would be :  

```
DISPLAY_BRAND_LOGO = SITEURL + "/images/logo.png"
```

Or

```
DISPLAY_BRAND_LOGO = "/images/logo.png"
```

### CODE REVIEW

**Comment by talha131** [comment]
Thank you @f-fouad I will merge it soon. My python setup is broken on my Mac machine (most probably because I am running beta macOS). As soon as it gets fixed, I will merge and release a new version of Elegant.

File: templates/base.html [hunk 1]
```diff
@@ -61,7 +61,12 @@
                             <span class="icon-bar"></span>
                             <span class="icon-bar"></span>
                         </a>
-                        <a class="brand" href="{{ SITEURL }}/"><span class=site-name>{{ SITENAME }}</span></a>
+                        <a class="brand" href="{{ SITEURL }}/">
+                            {% if DISPLAY_BRAND_LOGO %}
+                                <img src="{{ DISPLAY_LOGO_ON_TITLE }}" alt="brand-logo">
```
Comment (iranzo) [inline templates/base.html:66]: If Using DISPLAY_BRAND_LOGO the IMG SRC should be the same variable, if not, it sets an undefined variable

**Review by iranzo — CHANGES_REQUESTED** [review]

**Comment by iranzo** [comment]
Additionally I would consider creating an article (full article) with the feature to be added to the documentation. 

Let me know if you can work on this @f-fouad so that we can merge this. Great addition!

**Comment by iranzo** [comment]
Apart of the pelican logo, the variable used for enabling/disabling and the one used for 'showing' the actual icon are different ;-)

**Comment by iranzo** [comment]
@f-fouad are you going to write a small doc article regarding the feature?

**Comment by f-fouad** [comment]
Thank you for taking the time and effort to review it.

I am looking for an example of doc article right now (some guidance is welcome). Probably, it will take me some time because I am not accustomed to git/github (first pull request).

**Comment by iranzo** [comment]
It's a matter of adding a file like 'https://github.com/Pelican-Elegant/elegant/blob/master/documentation/content/Components/site-subtitle.md' but with 'site-logo' for example :) (and fill the contents there)

**Comment by iranzo** [comment]
Thank you!

**Pull request merged**